docs/index.rst

```diff
@@ -92,6 +92,20 @@ To obtain the most probable sequence of tags, use the `CRF.decode` method.
 
 This method also accepts a mask tensor, see `CRF.decode` for details.
 
+TorchScript
+-----------
+
+The ``CRF`` module is compatible with TorchScript on PyTorch ``>=1.10.0``.
+To get a ``torch.jit.ScriptModule``, wrap a ``CRF`` instance inside ``torch.jit.script``.
+
+.. code-block:: python
+
+   import torch
+   from torchcrf import CRF
+   num_tags = 5
```

Comment:
You can delete these lines because examples are assumed to continue, and these lines already exist in the very beginning.

Reply:
These are removed.